test/debugInfo.test.ts

```diff
@@ -0,0 +1,120 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the EPL v2.0 License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+import * as chai from 'chai';
+import * as chaipromise from 'chai-as-promised';
+import { ClientStubs } from './clientstubs';
+import { DebugInfo } from '../src/debugInfo';
+import { DebugInfoProvider } from '../src/debugInfoProvider';
```

Comment:
well, these files start with a capital D

Reply:
matches travis (build on mac succeeded while the one on linux failed). On Mac this doesn't matter while on Linux it does. (was caused by squashing, rebasing etc.)